modules/Departments/department.php

```diff
@@ -186,14 +173,8 @@
 
             //Print other courses
             if ($role == 'Coordinator' or $role == 'Assistant Coordinator' or $role == 'Teacher (Curriculum)' or $role == 'Teacher') {
-                $data = array('gibbonDepartmentID' => $gibbonDepartmentID, 'gibbonSchoolYearID' => $session->get('gibbonSchoolYearID'));
-                $sql = "SELECT gibbonSchoolYear.name AS year, gibbonCourse.gibbonCourseID as value, gibbonCourse.name AS name
-                        FROM gibbonCourse
-                        JOIN gibbonSchoolYear ON (gibbonCourse.gibbonSchoolYearID=gibbonSchoolYear.gibbonSchoolYearID)
-                        WHERE gibbonDepartmentID=:gibbonDepartmentID
-                        AND NOT gibbonCourse.gibbonSchoolYearID=:gibbonSchoolYearID
-                        ORDER BY sequenceNumber, gibbonCourse.nameShort, name";
-                $result = $pdo->executeQuery($data, $sql);
+                 
+                $result = $container->get(CourseGateway::class)->selectCourseListByOtherDepartment($gibbonDepartmentID, $session->get('gibbonSchoolYearID'));
```

Comment:
It looks like the intent of this code may be to get courses from past school years, so for consistency with the previous method, this name would make more sense:
```suggestion
                $result = $container->get(CourseGateway::class)->selectPastCoursesByDepartment($gibbonDepartmentID, $session->get('gibbonSchoolYearID'));
```